package.json

```diff
@@ -13,14 +13,27 @@
     "bower": "^1.7.7"
   },
   "devDependencies": {
+    "@bem/gulp": "0.0.5",
+    "@bem/gulp-bh": "gulp-bem/gulp-bh",
+    "autoprefixer": "^6.3.3",
     "bem": "^0.10.2",
+    "bem-config": "bem-contrib/bem-config#wip-new-with-rc",
     "borschik-tech-cleancss": "^2.1.0",
```

Comment:
it's just about consistence between `ENB` and `gulp-bem` (and reuse of same npm deps)